integrations/launchdarkly/.port/spec.yaml

```
@@ -0,0 +1,24 @@
+type: launchdarkly
+description: launchdarkly integration for Port Ocean
+docs: https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/launchdarkly
+icon: Launchdarkly
+features:
+  - type: exporter
+    section: Project management
+    resources:
+      - kind: project
+      - kind: flag
+      - kind: auditlog
+      - kind: environment
+configurations:
+  - name: launchdarklyHost
+    required: false
+    default: https://app.launchdarkly.com
+    type: string
+  - name: launchdarklyToken
+    required: true
+    sensitive: true
+    type: string
+  - name: appHost
+    type: url
+    required: false
```

Add description as well
```suggestion
  - name: launchdarklyToken
    required: true
    sensitive: true
    type: string
  - name: launchdarklyHost
    required: false
    type: string
    default: https://app.launchdarkly.com
  - name: appHost
    type: url
    required: false
```